src/api/routes/publisher.js

```diff
@@ -203,10 +236,75 @@ router.get('/:bbid/identifiers',
  */
 
 router.get('/:bbid/relationships',
-	makeEntityLoader('Publisher', relationshipsRelations, publisherError),
-	async (req, res) => {
+	makeEntityLoader('Publisher', utils.relationshipsRelations, publisherError),
+	async (req, res, next) => {
 		const publisherRelationshipList = await getEntityRelationships(res.locals.entity);
 		return res.status(200).send(publisherRelationshipList);
 	});
 
+/**
+ *	@swagger
+ * '/publisher':
+ *   get:
+ *     tags:
+ *       - Browse Requests
+ *     summary: Get list of Publishers which are related to an Edition or Work
+ *     description: Returns the list of Publishers, When one of the bbid of Work or Edition is passed as query parameter
+ *     operationId: getRelatedPublisherByBbid
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: work
+ *         in: query
+ *         description: BBID of the Work
+ *         required: false
+ *         type: bbid
+ *       - name: edition
+ *         in: query
+ *         description: BBID of the Edition
+ *         required: false
+ *         type: bbid
+ *     responses:
+ *       200:
+ *         description: List of Publisher with relationships which are related to either Work or Edition
+ *         schema:
+ *             $ref: '#/definitions/BrowsedPublishers'
+ *       404:
+ *         description: Work not found or Edition not found
+ *       406:
+ *         description: Invalid BBID paased in query params
+ */
+
+router.get('/',
+	formatQueryParameters(),
+	validateBrowseRequestQueryParameters(['author', 'edition', 'work', 'publisher']),
```

Comment:
As commented above, there is are extra special cases for links between Publisher and Edition that are not defined by relationships.
http://localhost:9098/1/publisher?edition=978c6f53-61e4-48cd-a75c-f220374215b3 returns an empty array where I would expect this entity to be returned: https://bookbrainz.org/publisher/9dc3a2f2-3d7d-4dda-b25b-b6de0de898fd

Reply:
Oh okayy. I didn't know this. i thought it was only for edition-editionGroup :sweat_smile:

Reply:
I too forgot about it !